terraform/fec_rds.tf

```diff
@@ -105,6 +105,11 @@ resource "aws_db_parameter_group" "fec_default" {
         name  = "log_statement"
         value = "ddl"
     }
+
+    parameter {
+        name  = "max_standby_streaming_delay" # This has no effect on masters, it only affects slaves
+        value = "1 h"
```

Comment:
Ah yes, the data type is `integer` for this, and the description in the RDS parameter settings says it's milliseconds:  `(ms) Sets the maximum delay before canceling queries when a hot standby server is processing streamed WAL data.`

So it looks like for this to be an hour, it needs to be set to `3600000`.  Just double checking:  is an hour appropriate here?